data/original/carriers.csv

```diff
@@ -2,6 +2,7 @@ id,name,tracking_url
 asendia,Asendia,https://track.aftership.com/asendia/
 canada-post,Canada Post,https://track.aftership.com/canada-post/
 chronopost,Chronopost,https://www.chronopost.fr/en/chrono_suivi_search?listeNumerosLT=
+deutsche-post,Deutsche Post,https://track.aftership.com/deutsch-post/
```

Comment:
is there a Deutsche Post URL for tracking?

Reply:
https://www.deutschepost.de/sendung/simpleQuery.html?locale=en_GB
This is the one I found out for Deutsche Post but not sure if it is correct.